src/common/Card.tsx

```diff
@@ -37,39 +38,37 @@ const Card: React.FunctionComponent<CardProps> = ({
   if (image) updatedImage = addImageTransformation(image);
 
   return (
-    <div className="card-demo pb-1">
-      <div className="card">
-        <div className="card__header">
-          {image ? (
-            <UserImage image={updatedImage} username={username} imageSize="xl">
-              <div className="avatar__intro">
-                <h4 className="avatar__name">{username}</h4>
-                <NavLink
-                  exact
-                  to={`/${id}/following`}
-                  activeClassName="bg-light btn-outline-secondary"
-                  className="btn text-primary btn-sm readings-sum"
-                >
-                  Following: {followings ? followings.length : 0}
-                </NavLink>
-                <span> </span>
-                <NavLink
-                  exact
-                  to={`/${id}/followers`}
-                  activeClassName="bg-light btn-outline-secondary"
-                  className="btn text-primary btn-sm readings-sum"
-                >
-                  Followers: {followers ? followers.length : 0}
-                </NavLink>
-              </div>
-            </UserImage>
-          ) : (
-            <h3>{username}</h3>
-          )}
-        </div>
-        <div className="card__footer">{children}</div>
-      </div>
-    </div>
+    <CardMUI>
+      <CardContent>
+        {image ? (
```

Comment:
Everything in this conditional statement is a part of the card header. Check out MUI's [Card Header](https://mui.com/api/card-header/#main-content) for this. Also, the [Avatar](https://mui.com/components/avatars/) and [Typography](https://mui.com/components/typography/) components will be helpful for the image and div, h4, and h3 html tags